docker-compose.yml

```diff
@@ -15,18 +15,18 @@ services:
       - "7777:7777"
     environment:
       - RUN_DEV_SERVER=True
-      - PORT=7777
-      - HOST=0.0.0.0
       - DEBUG=True
+      - PROTOCOL=https
+      - HOST=0.0.0.0
+      - PORT=7777
       - VERSION_PATH=/version.json
       - SECRET_KEY=secret_key_change_me
-      - SESSION_COOKIE_NAME=lando-ui.dev
-      - SESSION_COOKIE_DOMAIN=
-      - SESSION_COOKIE_SECURE=False
+      - SESSION_COOKIE_NAME=lando.mozilla.org
+      - SESSION_COOKIE_DOMAIN=lando.mozilla.org
+      - SESSION_COOKIE_SECURE=1
```

Comment:
I see that it does default to true farther down.  I think you can drop this line and rely on the default.